lib/PuppeteerSharp.Tests/ElementHandleTests/ClickTests.cs

```diff
@@ -34,8 +34,8 @@ public async Task ShouldWorkForShadowDomV1()
         public async Task ShouldWorkForTextNodes()
         {
             await Page.GoToAsync(TestConstants.ServerUrl + "/input/button.html");
-            var buttonTextNode = (await Page.EvaluateExpressionHandleAsync(
-                "document.querySelector('button').firstChild")) as ElementHandle;
+            var buttonTextNode = (ElementHandle) await Page.EvaluateExpressionHandleAsync(
```

Comment:
no null check after an `as` implied it can only be that type. hence a cast is better, since if the assumption is incorrect you will get a useful cast exception instead of a useless null ref exception